### ISSUE TITLE
Running out of space on ImageKit

### ISSUE THREAD

**Issue by perfectly-preserved-pie** [perfectly-preserved-pie/larentals#193]
I'm running out of space on the ImageKit free plan:
![image](https://github.com/perfectly-preserved-pie/larentals/assets/28774550/c7d1a2ae-66a0-40cf-9059-39cb100dccfb)

Luckily when I wrote the ImageKit code I was smart enough to upload each image with the filename set to the MLS number as a unique identifier so I can iterate through the dataframes with this logic:

If _MLS number_ doesn't exist in the _list of ImageKit filenames_, remove the _ImageKit file_

Basically doing a comparison between active listings and what I have in ImageKit. If the item doesn't exist in the dataframe but does exist in ImageKit, then it's an inactive listing and we can remove it from ImageKit to free up space.
